zng/docs/zson.md

```diff
@@ -506,52 +536,76 @@ map type, or two elements referring to the type of the keys and type of the valu
 A map value may be empty.  An empty map value without a type decorator is
 presumed to be an empty map of type (`null`, `null`).
 
-### 3.5 Type Value
+#### 3.4.7 Type Value
+
+The type of a type value is `type` while its value is depicted by parenthesizing
+its type description (as defined in [Section 3.5](#35-type-syntax).  For example,
+a value of a record with a single field called `t` of type `type` might look
+like this:
+```
+{ t: (string) (type) }
+```
+Since type values have implied types, the `type` type decorator can be omitted:
+```
+{ t: (string) }
+```
+Now supposing we created a second field called `t2` whose type is
+computed by introspecting the type of `t`.  This result is
```

Comment:
```suggestion
Now suppose we added a second field called `t2` whose type is
computed by introspecting the type of `t`.  The resulting record type value is
```

Reply:
(I think if I were reading this without knowing about the latest zql, I might now be stuck wondering "what do they mean by "introspecting" here?)

Reply:
Fair enough, but "type introspection" is a well-known concept from the world of programming languages and compilers.

Reply:
Totally! I would be intrigued and wondering "how do i introspect this stuff" 🤓